src/interface_adapter/vote_out/VoteOutController.java

```diff
@@ -1,4 +1,17 @@
 package interface_adapter.vote_out;
 
+import use_case.vote_out.VoteOutInputBoundary;
+import use_case.vote_out.VoteOutInputData;
+
 public class VoteOutController {
+    final VoteOutInputBoundary userVoteOutUseCaseInteractor;
```

Comment:
insert a space between class attributes and class methods for better readability